post-processing/config_handler.py

```diff
@@ -280,8 +297,12 @@ def read_config(config: dict):
 
     # check optional series information
     if config.get("series"):
-        if len(config.get("series")) == 1:
-            raise RuntimeError("Number of series must be >= 2.")
+        if plot_type == "generic":
+            if len(config.get("series")) == 1:
+                raise RuntimeError("Number of series must be >= 2 for generic plot.")
+        if plot_type == "line":
```

Comment:
Why would this be different for line plots?
It looks like it makes more sense to have `len(series)>=1` instead of 2, but that should be investigated in a different issue. Here just keep the same limit for both types of plots (if nothing breaks)

Reply:
The test for series probably needs changing as well.